controller/ovn-controller.c

```diff
@@ -2562,6 +2564,18 @@ check_northd_version(struct ovsdb_idl *ovs_idl, struct ovsdb_idl *ovnsb_idl,
     return true;
 }
 
+static bool print_records_callback(uintptr_t arg, const ddlog_record *rec, ssize_t weight) {
+    VLOG_INFO("Printing DDlog records\n");
```

Comment:
Usually we don't write `\n` at the end of a log message, because the log system does that for us. (It doesn't hurt anything though.)